tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# the inclusion of the tests module is not meant to offer best practices for
+# testing in general, but rather to support the `find_packages` example in
+# setup.py that excludes installing the "tests" package
```

Comment:
@qwcode Originally I intended it to include a good starting point for tests, and I would have happily included docs if I'd known how to (:-)) but as the recent thread shows, we can't even suggest an approach for testing without getting into trouble :-(

But if your plan is to keep tests and just go with an example which (as per the comment you modified below) people can ignore if they don't like, then I guess I'm fine with it. I'm certainly happier to *have* a sample approach to testing than not, even if we have to add disclaimers. (I'd be OK with doing the same for docs, but that's a different PR).